### ISSUE TITLE
False positive for abstract-method (W0223)

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#155]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

Pylint does not detect overriding of abstract attributes by non-function objects.  For example, in the following code, the definition of E (but not the one of D) raises the warning "Method 'prop' is abstract in class 'C' but is not overridden (abstract-method)".

```
#!python
import abc

class C(metaclass=abc.ABCMeta):
    @property
    @abc.abstractmethod
    def prop(self):
        pass

class D(C):
    @property
    def prop(self):
        return 1

class E(C):
    prop = property(lambda self: 1)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/155
